submissions/LuckyDnepr/Hooli-popup-task/styles-menu.css

```diff
@@ -0,0 +1,304 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
+
+html {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+a {
+  text-decoration: none;
+  font-size: 16px;
+  color: black;
+}
+
+.wrapper {
+  display: grid;
+  width: 80%;
+  min-width: 320px;
+  min-height: 100vh;
+  margin: auto;
+  grid-template-columns: 100%;
+  grid-template-rows: 100px auto 70px;
+  justify-content: center;
+  align-content: stretch;
+  justify-items: center;
+  align-items: center;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  width: 100%;
+  justify-self: end;
+  align-self: center;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+.logo {
+  display: block;
```

Comment:
Great Job!  😁

Lets make some small changes:

You apply class `logo` for `div` tag. `div` is block element by default.  

[Please read](https://www.w3schools.com/html/html_blocks.asp)